contracts/TalentLayerService.sol

```diff
@@ -299,14 +372,16 @@ contract TalentLayerService is Initializable, ERC2771RecipientUpgradeable, UUPSU
      * @param _platformId platform ID on which the Service token was created
      * @param _dataUri IPFS URI of the offchain data of the service
      * @param _signature optional platform signature to allow the operation
+     * @param _token token address to be used for the service's payments
      */
     function createService(
         uint256 _profileId,
         uint256 _platformId,
         string calldata _dataUri,
-        bytes calldata _signature
+        bytes calldata _signature,
+        address _token
```

Comment:
as there is a breaking change here, we can still have one only createService function